source/delegate/test/Delegate-unit.js

```diff
@@ -318,13 +318,13 @@ contract('Delegate Unit Tests', async accounts => {
 
   describe('Test setRuleAndIntent()', async () => {
     it('Test calling setRuleAndIntent with transfer error', async () => {
-      let intentAmount = 250
+      let stakedAmount = 250
```

Comment:
Being picky here... I'd probably prefer it as `stakeAmount` or `stakingAmount` or something. `staked` to me implied it has already happened which it hasnt. Might be being too picky though

Reply:
done.